inception-doc/src/main/resources/META-INF/asciidoc/user-guide/getting-started.adoc

```diff
@@ -1,79 +1,399 @@
+to simplicity and focus on the first steps. For further reading on any
+issue, see our main documentation.: LINK EINFÜGEN. For quick overviews,
+also see our tutorial videos (Topics: Introduction, Overview,
+Recommender Basics, Entity Linking):
+https://www.youtube.com/playlist?list=PL5Hz5pttaj96SlXHGRZf8KzlYvpVHIoL-
 
-In order to run {product-name}, you need to have Java installed on your system in version 8 or 
-higher. If you do not have Java installed yet, please the latest link:https://www.oracle.com/technetwork/java/javase/downloads/index.html[Oracle Java] or link:https://adoptopenjdk.net[OpenJDK].
+ +
+_Introduction_
 
-Download the stand-alone JAR from the link:https://inception-project.github.io/downloads/[{product-name} downloads page]. 
+* What can INCEpTION be used for? (also see our introduction video for a
+first impression what INCEpTION is):
+https://www.youtube.com/watch?list=PL5Hz5pttaj96SlXHGRZf8KzlYvpVHIoL-&time_continue=135&v=Ely8eBKqiSI&feature=emb_logo)
 
-Start the application simply by *double-clicking* on the download JAR file in your file manager.
-After a moment, a splash screen will be displayed while the application is initializing. Once the
-initialization is complete, a dialog will appear which you can use to open the application in your
-default browser or to shut down the application.
+* {blank}
+** INCEpTION is a text-annotation enviroment useful for various kinds of
+annotation projects. The core functionality is annotating any kind of
+written text. Annotations can be used both for linguistic and machine
+learning concerns.
+** But INCEpTION is not only an annotation tool. It integrates three
+steps in one plattform: creating a corpus to be annotated (*Corpus
+Creation*); annotating it and merging annotations made by several
+annotators (*Annotation, Curation*); and managing the knowledge used for
+annotating (*Knowledge Bases*.) You may want to do one, two or all three
+steps with INCEpTION. Also, it is extendable and adoptable to individual
+requrements in all of these three areas. This Guide focuses on
+Annotation. For details on Corpus Creation and Knowledge
+Management/Knowledge Bases, see the main documentation.
++
+* Tell us what you would like to have
+** INCEpTION is still in developement, so you are welcome to give us
+feedback and tell us your wishes and requirements. For any questions,
+you are welcome to join our Google group „inception users“:
+https://groups.google.com/forum/#!forum/inception-users
++
+* Know our documentation for further reading
+** You will find lots of information in our Guides, called documentation
+(on our website, among the tabs on top: → _Documentation:_
+https://inception-project.github.io/documentation/[https://inception-]https://inception-project.github.io/documentation/[project.github.io/documentation/]
+. There, see the upper section.). If they do not answer your question,
+don‘t hesitate to contact our team: ____ We are glad to help you.
```

Comment:
google group or github

Reply:
I added "see above" since the contact ist 6 lines above.